site-scrapers/LowellGeneral.js

```diff
@@ -0,0 +1,168 @@
+const sites = require("../data/sites.json");
+const https = require("https");
+const html_parser = require("node-html-parser");
+
+const siteName = "Lowell General";
+const site = sites[siteName];
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${siteName} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${siteName} done.`);
+    return webData;
+};
+
+function leadZero(num) {
+    return num < 10 ? "0" + num : num;
+}
+
+function buildTimeSlotsUrl(stationId) {
+    let doctorId = stationId.split("_")[6];
+    let locationId = stationId.split("_")[7];
+    let stationNumber = stationId.split("_")[1].replace("appointments", "");
+    let now = new Date();
+    let startDate =
+        [
+            now.getFullYear(),
+            leadZero(now.getMonth() + 1),
+            leadZero(now.getDate() + 1),
```

Comment:
When run in a lambda I think this will return the day in the GMT time zone, which could be off from Eastern time. I would guess that the request parameters are expected to be in eastern time.

Reply:
The time of day is zero'd out ` +00%3A00%3A00` but I suppose if it's UTC that would mean the getDate would return the next day. I could do getDate() - 1 to be safe. The plus 1 was a copy paste error.